pyramid_oauthlib/__init__.py

```diff
@@ -11,14 +11,18 @@
 
 OAUTH_PARAMS = (
     'access_token',
+    'client',
```

Comment:
I don't think `client` is a known parameter in the spec. Is it in popular use by some client library?

https://tools.ietf.org/html/rfc6749#section-11.2.2

Reply:
I will check if I actually use it. I wonder if I pulled it from an example somewhere.